File: BubbleSort.java
//Bubble Sort in java
//Bubble Sort, sort the array in ascending order by repeatedly swapping the adjacent elements if they are in wrong order
import java.util.Arrays;
public class BubbleSort{
	public static void main(String[] args){
		int[] arr={4,2,9,-1,0};
		bubbleSort(arr);
		System.out.println(Arrays.toString(arr));
	}
	static void bubbleSort(int[] arr){
		for(int i=0;i<arr.length;i++){
			for(int j=1;j<arr.length-i;j++){
				if(arr[j-1]>arr[j]){
					int temp=arr[j-1];
					arr[j-1]=arr[j];
					arr[j]=temp;
				}
			}
		}
	}
}

File: BubbleSort.cpp
//Bubble sort in c++
#include<iostream>
using namespace std;
void sort(int arr[],int len){
    for(int i=0;i<len;i++){
        for(int j=1;j<len-i;j++){
            if(arr[j-1]>arr[j]){
                int temp=arr[j];
                arr[j]=arr[j-1];
                arr[j-1]=temp;
            }
        }
    }
    for(int i=0;i<len;i++){
        cout<<arr[i]<<" ";
    }
}
int main(){
    int arr[]={1,2,-1,-9,0,22};
    int len=sizeof(arr)/sizeof(arr[0]);
    sort(arr,len);
    return 0;
}